CHANGELOG.md

```diff
@@ -7,18 +7,18 @@
 * Added an optional `fetch` parameter to the `AppConfiguration`. Use this to specify a custom implementation of the `fetch` function used by the app to perform network requests.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None
+* Handle `EOPNOTSUPP` when using `posix_fallocate()` and fallback to manually consume space. This should enable android users to open a Realm on restrictive file systems. ([#6349](https://github.com/realm/realm-js/issues/6349), since v12.4.0)
```

Comment:
In the issue is said that it happened after upgrading to `v12.2.0`.

Reply:
You have a point, and the core change log actually says "... also an issue in some platforms before this".